week2/homework/src/index.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+'use strict';
+// const program = require('commander');
+
+const command = process.argv[2];
+const argument = process.argv.slice(3);
+const renderList = require('./todolist');
+const addToList = require('./addtodo');
+const removeToDo = require('./removetodo');
+const reset = require('./reset');
+const update = require('./update');
+const helpCommand = require('./help');
+function actions(commandsCallFunc) {
+  commandsCallFunc.then(function() {
+    return renderList();
+  });
+}
+
+if (command === 'list') {
+  renderList();
+}
+ else if (command === 'add') {
+  actions(addToList(...argument));
+}
+ else if (command === 'remove') {
+  if (argument < 0) {
+    console.log('index must be greater than 0');
+  }
+  actions(removeToDo(argument));
```

Comment:
Maybe you don't want to call the function if doesn't meet your condition? So `else { actions(removeToDo(argument)); // .. `